lte/gateway/c/core/oai/tasks/amf/amf_app_state_manager.cpp

```diff
@@ -176,21 +193,24 @@ map_uint64_ue_context_t* AmfNasStateManager::get_ue_state_map() {
 // This is a helper function for debugging. If the state manager needs to clear
 // the state in the data store, it can call this function to delete the key.
 void AmfNasStateManager::clear_db_state() {
+  OAILOG_FUNC_IN(LOG_AMF_APP);
   OAILOG_DEBUG(LOG_AMF_APP, "Clearing state in data store");
   std::vector<std::string> keys_to_del;
   keys_to_del.emplace_back(AMF_NAS_STATE_KEY);
 
   if (redis_client->clear_keys(keys_to_del) != RETURNok) {
     OAILOG_ERROR(LOG_AMF_APP, "Failed to clear the state in data store");
-    return;
+    OAILOG_FUNC_OUT(LOG_AMF_APP);
   }
+  OAILOG_FUNC_OUT(LOG_AMF_APP);
 }
 
 void put_amf_ue_state(amf_app_desc_t* amf_app_desc_p, imsi64_t imsi64,
                       bool force_ue_write) {
+  OAILOG_FUNC_IN(LOG_AMF_APP);
   if ((!AmfNasStateManager::getInstance().is_persist_state_enabled()) ||
       (imsi64 == INVALID_IMSI64)) {
-    return;
+    OAILOG_FUNC_OUT(LOG_AMF_APP);
```

Comment:
Add error log with message

Reply:
It is not an error condition. The UE state is synced if imsi64 is valid.